packages/frenetic/frenetic.3.4.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+ocaml-version: [ >= "4.01.0" ]
+version: "3.4.0"
+maintainer: "Arjun Guha <arjun@cs.umass.edu>"
+build: [
+  ["./configure" "--%{pa_ounit:enable}%-tests" "--prefix" prefix ]
+  [make]
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "netkat"]
+]
+depends: [
+  "ocamlfind"
```

Comment:
I think the difference is you wouldn't necessarily want frenetic to recompile if ocamlfind is updated.